realm/realm-library/src/androidTest/java/io/realm/entities/IOSAllTypes.java

```diff
@@ -31,14 +31,12 @@
     private boolean boolCol;
     private short shortCol;
     private int intCol;
-    private long longCol;
+    private int longCol;
+    private long longLongCol;
     private float floatCol;
     private double doubleCol;
-    @Required
     private byte[] byteCol = new byte[0];
-    @Required
     private String stringCol = "";
-    @Required
```

Comment:
@beeender I just realized that there is no `null` test for blob and string. (... aren't they same anyway?) I'll add those test first thing in the morning.